pydis_site/apps/content/resources/guides/python-guides/subclassing_context.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Subclassing Context
+description: "Subclassing the default commands.Context to add more functionability and customisability."
+---
+
+## Basic Subclassing
+First, the [documentation on inheritance](https://docs.python.org/3/tutorial/classes.html#inheritance) on subclassing will provide some fundamental knowledge, which is highly suggested before moving on to this topic, as subclassing [Context](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Context) can ultimately be a complicated task.
+
+## The benefits of subclassing Context
+Subclassing Context can be very beneficial as it allows you to set custom Context methods that can be used in your code. Some applications and examples are provided below.
+
+In order to subclass Context, you are required to subclass `commands.Bot` as well. More on subclassing Bot can be read here [link placeholder] but here is a short example.
```

Comment:
Isn't it easier to only mention subclassing the Bot class? You don't need to link to that first documentation, since that is done in the bot one.